velox/dwio/dwrf/writer/WriterContext.h

```diff
@@ -182,6 +182,10 @@ class WriterContext : public CompressionBufferPool {
 
   int64_t getMemoryUsage(const MemoryUsageCategory& category) const;
 
+  std::string getWriterMemoryPoolStats() {
```

Comment:
It this for test only? And getMemoryUsage already returns the actual used memory? thanks!